docs/nidmm/functions.rst

```diff
@@ -703,7 +703,7 @@ nidmm.Session methods
         .. note:: One or more of the referenced values are not in the Python API for this driver. Enums that only define values, or represent True/False, have been removed.
 
 
-    :type maximum_time: int
+    :type maximum_time: float in seconds or datetime.timedelta
```

Comment:
These are significant changes worth noting in CHANGELOG because they are silent source breakers.
Or is this simply a documentation bug getting addressed?

Reply:
This was a metadata bug being addressed. nidmm & niswitch were still using `'python_api_converter_type'` instead of `'python_type'`. Now all drivers are using `'type_in_documentation'`